phys2bids/phys2bids.py

```diff
@@ -264,6 +265,10 @@ def phys2bids(filename, info=False, indir='.', outdir='.', heur_file=None,
 
     if heur_file and sub:
         LGR.info(f'Preparing BIDS output using {heur_file}')
+        # Generate participants.tsv file if it doesn't exist already.
+        # Update the file if the subject is not in the file.
+        # Do not update if the subject is already in the file.
+        participants_file(indir, outdir, yml, sub)
```

Comment:
What if we remove `indir` and just ask for the absolute path to the yaml file?

Reply:
In fact, in order to simplify the arguments of the CLI, we could remove `indir` for good...

Reply:
Sure! Sounds good.

Reply:
We can do that in another PR.